scripts/postinstall.js

```diff
@@ -0,0 +1,40 @@
+const { promisify } = require('util')
+const fs = require('fs')
+const path = require('path')
+
+const copyFile = promisify(fs.copyFile)
+const mkdir = promisify(fs.mkdir)
+
+ensureEmptyDotPrisma()
+
+async function ensureEmptyDotPrisma() {
+  try {
+    const inTheNodeModules =
+      __dirname.endsWith('/node_modules/nexus-prisma/scripts') ||
+      __dirname.endsWith('/.yalc/nexus-prisma/scripts')
+    const dotNexusPrismaDir = path.join(
+      __dirname,
+      inTheNodeModules ? '../../../node_modules/.nexus-prisma' : '../.nexus-prisma'
```

Comment:
Is there a safer way to locate node modules than by doing this relative traversal?